src/Http/MetricsController.php

```diff
@@ -1,11 +1,11 @@
 <?php
 
-namespace Valentin\Mojam\Http;
+namespace ShaitanMasters\Prometheus\Http;
 
 use Illuminate\Contracts\Routing\ResponseFactory;
 use Illuminate\Routing\Controller;
 use Illuminate\Http\Response;
-use Valentin\Mojam\Exporters\PrometheusExporter;
+use ShaitanMasters\Prometheus\Exporters\PrometheusExporter;
 
 class MetricsController extends Controller
 {
```

Comment:
`return $this->responseFactory->make($metrics, 200, ['Content-Type' => PrometheusExporter::MIME_TYPE]);`
lets make 200 not a number but Response const